wpeverest/user-registration
# Enhance - Core Login disable feature QoL changes

### All Submissions:






### Changes proposed in this Pull Request:



Closes #207 .

### How to test the changes in this Pull Request:

1. Add an invalid page as the my account page.
2. Try navigating to the wp-admin page.
3. Observe that the my account page shows up if the correct param for page is present, otherwise see that it allows the navigation to wp-admin.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Core Login disable feature QoL changes.
